pulpcore/cli/ansible/content.py

```diff
@@ -0,0 +1,29 @@
+import gettext
+
+import click
+
+from pulpcore.cli.ansible.context import PulpAnsibleCollectionVersionContext
+from pulpcore.cli.common.context import PulpContext, pass_pulp_context
+from pulpcore.cli.common.generic import list_command
+
+_ = gettext.gettext
+
+
+@click.group()
+@click.option(
+    "-t",
+    "--type",
+    "content_type",
+    type=click.Choice(["collection-version"], case_sensitive=False),
+    default="collection-version",
+)
+@pass_pulp_context
+@click.pass_context
+def content(ctx: click.Context, pulp_ctx: PulpContext, content_type: str) -> None:
+    if content_type == "collection-version":
+        ctx.obj = PulpAnsibleCollectionVersionContext(pulp_ctx)
+    else:
```

Comment:
Any chance we can add the code path for roles right now? Also OK to do it later.